### ISSUE TITLE
Fans won't come on

### ISSUE THREAD

**Issue by 53245342099** [Rem0o/FanControl.Releases#898]
I really don't know where to begin.  I got a new graphics card, and after I put it in, Fan Control still had the old one as a label.  I couldn't immediately figure out how to start a brand new config, so I just deleted Fan Control and started over.  I did the automated set up and when I was supposed to identify fans I noticed a problem: the bottom front fan wouldn't spin.  Couldn't figure out why.  Deleted Fan Control and started over.  This time things seemed to go okay.  Go to set my curves and even though the rear fan was spinning only very slowly, Fan Control reported that it was spinning at 2300 RPM.  Okay.  Start over again. This time when identifying fans, the rear wouldn't come on at all.  Start over again and everything seemed to be okay, but then I noticed that the bottom front fan wasn't spinning.  Fool around with it...changed it from curve to manual.  It began to spin.  Then I go to stress the CPU and noticed the rear fan wasn't spinning.  Got it to spin by fooling around with it, but now it has stopped again.  I don't know why.  Don't get it at all.  Is it the software?  Did I screw something up when I changed graphics cards?  

Thanks for any assistance troubleshooting this.

**Comment by 53245342099** [comment]
A better title for this might be "fans turn off".  I can get them to come on if I choose "Detect fan start % automatically".  It works for a while then it will turn off again.  Can happen to either the rear fan or the lower front fan.

**Comment by 53245342099** [comment]
https://i.imgur.com/B7VdrNF.png

Front top didn't come on until CPU temp was 60+, and it didn't come on immediately.  Rear fan still hasn't come on.  They were on earlier but they quit and I have a hard time getting them going again.

**Comment by Rem0o** [comment]
Let me guess, Asus motherboard?

**Comment by 53245342099** [comment]
Nah, MSI:  https://www.msi.com/Motherboard/MAG-Z690-TOMAHAWK-WIFI-DDR4

After opening the case for the third time and wiggling wires around, it seems to have resolved itself at least for now.  I couldn't tell anything was loose.
